modules/swagger-codegen/src/main/resources/python/api_client.mustache

```diff
@@ -176,14 +176,20 @@ class ApiClient(object):
         :param obj: The data to serialize.
         :return: The serialized form of data.
         """
-        types = (str, float, bool, tuple) + tuple(integer_types) + (text_type,)
+        types = (str, float, bool, bytes) + tuple(integer_types) + (text_type,)
         if isinstance(obj, type(None)):
             return None
         elif isinstance(obj, types):
             return obj
         elif isinstance(obj, list):
             return [self.sanitize_for_serialization(sub_obj)
                     for sub_obj in obj]
+        elif isinstance(obj, tuple):
+            if len(obj) == 1 and isinstance(obj[0], list) and len(obj[0]) == 3:
```

Comment:
I don't think this should be left in, it is not necessary for handling multipart files and could lead to unintended behaviour in the unlikely chance a list had this structure and it wasnt a file